content/manuals/harmonia/_index.md

```diff
@@ -1,26 +1,28 @@
 ---
-title: Docker Run Cloud
-description: Learn how you can run your applications in the cloud with Docker Run Cloud
+title: Project Harmonia
+description: Learn how you can run your applications in the cloud with Project Harmonia
 keywords: run, cloud, docker desktop, resources
 sitemap: false
+aliases:
+- /run-cloud/
 ---
 
 {{% restricted title="Private preview" %}}
-Docker Run Cloud is in Private preview.
+Project Harmonia is in Private preview.
 {{% /restricted %}}
 
-Docker Run Cloud brings the power of the cloud to your local development workflow. You can now run your applications in the cloud whilst continuing to use your existing tools and workflows and without worrying about local resource limitations. Docker Run Cloud also lets you share previews of your cloud-based applications for real-time feedback. 
+Project Harmonia brings the power of the cloud to your local development workflow. You can now run your applications in the cloud whilst continuing to use your existing tools and workflows and without worrying about local resource limitations. Project also lets you share previews of your cloud-based applications for real-time feedback. 
```

Comment:
"Project also lets you share.."

Should this be "Project Harmonia also let you share..."